docs/docs/adding-page-transitions-with-plugin-transition-link.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Using transition-link for Page Transitions
+---
+
+This guide will cover how to use `gatsby-plugin-transition-link` to animate transitions between pages on your Gatsby site.
+
+## Overview
+
+The `TransitionLink` component is a simple way of describing a page transition via props on a Link component. It works with many animation libraries, like [react-pose](https://popmotion.io/pose/), [gsap](https://greensock.com/), [animejs](https://animejs.com/), and many others.
```

Comment:
Let's avoid using the word "simple" here, as it might not be simple to everyone. Perhaps it could be reworded to say something like this:
"The `TransitionLink` component provides a way to describe a page transition via props on a Link component."